### PR TITLE
Move upstream to maven

### PR DESCRIPTION
**Pull request by pjgg**

### CODE REVIEW

File: backend/petshop/pom.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <name>Micifuz: petshop</name>
+    <dependencies>
+        <dependency>
+            <groupId>com.micifuz</groupId>
+            <artifactId>commons</artifactId>
+        </dependency>
+        <!-- Test resources -->
+        <dependency>
+            <groupId>com.micifuz</groupId>
+            <artifactId>test-resources</artifactId>
```
Comment (gutmox) [inline backend/petshop/pom.xml:21]: Should test-resources be  under test scope?

File: backend/shelters/shelters.iml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gutmox) [inline backend/shelters/shelters.iml:1]: Did you want to commit this ij file?

File: backend/vets/vets.iml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gutmox) [inline backend/vets/vets.iml:1]: Did you want to commit this ij file?

**Review by gutmox — CHANGES_REQUESTED** [review]
minor comments

File: backend/petshop/pom.xml (hunk 1 above)
Comment (pjgg) [reply backend/petshop/pom.xml:21]: the scope has moved to dependency management

```
<dependency>
                <groupId>com.micifuz</groupId>
                <artifactId>test-resources</artifactId>
                <version>${project.version}</version>
                <scope>test</scope>
            </dependency>
```

**Review by gutmox — APPROVED** [review]

**Pull request merged**